src/shiv/pip.py

```diff
@@ -68,6 +59,10 @@ def install(interpreter_path: str, args: List[str]) -> None:
     """
     with clean_pip_env():
 
+        # convert '.' to absolute path if it exists
+        if '.' in args:
```

Comment:
this will break if someone uses `..`. Not sure how comprehensive you want to be. Maybe:

```python
# convert '.' to absolute paths if it exists
for i, arg in enumerate(args):
    if arg.startswith('.'):
        args[i] = Path.cwd().joinpath(arg).absolute().as_posix()
```